public/_headers

```diff
@@ -7,3 +7,8 @@
 
 /*
   Content-Security-Policy: CSP-DIRECTIVES-PLACEHOLDER;
+  X-frame-options: sameorigin
+  X-XSS-Protection: 1; mode=block
+  X-Content-Type-Options: nosniff
+  Referrer-Policy: strict-origin-when-cross-origin;
+  Feature-Policy: payment 'self'; geolocation 'self'
```

Comment:
The example app is the foundation for new scrivito apps. I can't foresee which features the developer wants to use or how she/he will use them. E.g. if someone will incorporate a payment iframe or a payment widget.

Therefore I suggest to just remove the `Feature-Policy` since I don't see a good default value, that will not hinder our customers, if they try to implement such a feature.

Or we could simply set it empty:

```
Feature-Policy:
```

This way we would still pass the security headers test:

> To further the exposure of the header I will be adding detection to Security Headers. The header will not have a weight to impact the grade but it will be flagged as required. I can start to track utilisation and make people aware of its existence to drive adoption!

Source: https://scotthelme.co.uk/a-new-security-header-feature-policy/